src/redux/categories/categories.js

```diff
@@ -0,0 +1,40 @@
+// Initial category
+const initialCategoryState = [];
+
+// create action contants - This are calleds action types
+const ADD_CATEGORY = 'category/ADD_CATEGORY';
+const STATUS_CHECKED = 'category/STATUS_CHECKED';
```

Comment:
- [ ] Kindly be noticed that you should only create the `CHECK_STATUS` action and the value of the action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`.